HomeWork/src/main/java/io/FileLoggerConfiguration.java

```diff
@@ -0,0 +1,105 @@
+package io;
+
+import java.io.File;
+import java.io.IOException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+public class FileLoggerConfiguration {
+
+    String fileName = "NIO.txt";
+    private LoggingLevel loggingLevel;
+    private LoggingLevel[] arrayLevels;
+    protected int maxSizeFile = 1024;
+    private String formatFile = " [ПОТОЧНИЙ_ЧАС][LEVEL] ";
+
+    protected File logFile = new File(fileName);
+
+    public FileLoggerConfiguration() {
+    }
+
+    public FileLoggerConfiguration(String fileName, LoggingLevel loggingLevel, int maxSizeFile, String formatFile) {
+        if (LoggingLevel.DEBUG.equals(loggingLevel)) {
+            arrayLevels = new LoggingLevel[2];
+            arrayLevels[0] = LoggingLevel.INFO;
+            arrayLevels[1] = LoggingLevel.DEBUG;
+        } else {
+            arrayLevels = new LoggingLevel[1];
+            arrayLevels[0] = LoggingLevel.INFO;
+        }
```

Comment:
This code must be removed as it is redundant. You should initialize 4 fields here in constructor - `fileName, loggingLevel, maxFileSize, format`